doc/2/protocols/websocket/constructor/index.md

```diff
@@ -0,0 +1,43 @@
+---
+code: true
+type: page
+title: Constructor
+description: Creates a new WebSocket protocol
+order: 50
+---
+
+# Constructor
+
+Initializes a new instance of the WebSocket class pointing to the Kuzzle server specified by the uri.
+
+## Arguments
+
+```dart
+WebSocketProtocol(
```

Comment:
(open to debate) shouldn't this constructor use a [Uri object](https://api.dart.dev/stable/2.8.4/dart-core/Uri-class.html) instead of a host argument + port & ssl options?

Reply:
Yeah I could use an Uri object (note that the ssl option is not part of this object so I will have to leave it as an option in the ctor)

Reply:
Done :+1: